custodia/httpd/server.py

```diff
@@ -414,4 +423,7 @@ def get_socket(self):
         return (self.httpd.socket, self.httpd.socket_file)
 
     def serve(self):
+        if sd_notify is not None and systemd_booted():
+            logger.debug('sd_notify("READY=1")')
```

Comment:
What's the point of this log statement ?
I do not think we really care to log a functional detail of the startup process, systemd is already loggin that for us.